chapt1.md

```diff
@@ -2,6 +2,8 @@
 
 My suggestion that we think of bible as an anarchic interface has two main aims. First, bible as interface can challenge ideas and practices that reduce bible to simply a container of content to be consumed and can guide our imagination of bible in a time of major media transition. Second, an interface that resists our recourse to an original text or singular governing principle in the act of reading can help focus us on the vast productive potentialities of this interface even as it is translated into media technologies we have yet to imagine. This first chapter will focus on the first of these aims by advocating for a concept^[Might add a note here from Piotr about the preference for concept over definition.] of interface as a space and a process that facilitate relationship without eradicating difference. 
 
+How about inserting a section here on bible as container related to canonicity and such, or maybe a section on Plate's skinscape?
```

Comment:
But, could it be that the connection between skinscape and media is a way toward locating interface within the discourse of material religion?